Jenkinsfile

```diff
@@ -0,0 +1,82 @@
+pipeline {
+    agent none
+    environment {
+        projectName = 'DDF'
+    }
+    stages {
+        stage('Setup') {
+            steps{
+                notifySlack('good', "STARTED: ${JOB_NAME} ${BUILD_NUMBER} ${BUILD_URL}")
+            }
+        }
+        stage('Parallel Build') {
+            //TODO Parallel Stage Syntax may soon be changing for declaritive Jenkinsfiles - https://issues.jenkins-ci.org/browse/JENKINS-41334
```

Comment:
Good find on this ticket!

Since I had some confusion understanding what you meant, this TODO could be improved to something like
> // TODO refactor this stage from scripted syntax to declarative syntax to match the rest of this file - https://issues.jenkins-ci.org/browse/JENKINS-41334